shopsdk/orderprintful.py

```diff
@@ -0,0 +1,120 @@
+import json as a
+import pickle as b
+import ntplib as am
+import printful as e
+import pprint as i
+import pyminizip as g
+import digitalcontentsender as ar
+import datadog as an
+import utilsnotbyme as ao
+def orderprintful(k, l, m, n, o, p, q, r, s, t, u, v, w, ao, ap):
+  ae='rb'
+  with open("./config.json", ae) as f:
+    j = a.load(f)
+    aa=j["printful_production"]
+    ab=j["printful_send_logs"]
+    ac=j["printful_use_pickle"]
+    ad=j["printful_execute_all"]
+    af=j["offset.dat"]
+  if aa==true and ab==true and j["printful_load_options_file"] and ac==true:
+    y = open(options.dat, ae)
+    options = b.load(y)
+  if aa==true and ab==true and ac==true:
+    ao.setup_log(j["printful_sentry_url"], False)
+  if aa==true and j["printful_use_private_key"] and ad==true:
+    x = open(password.dat, 'rb')
+    ag = b.load(x)
+    w = str(j.getcwd())
+    v = j["printful_cdn_url"] + "/" + k
+    g.uncompress("offset.enc", v, w, int(withoutpath))
```

Comment:
What does this do? What is it intended to accomplish?

Reply:
That actually can be removed, I did that because I ran out of a use for something. I'll get rid of that when I do the digicam print channel code.